src/dmd/mtype.d

```diff
@@ -6521,3 +6459,80 @@ const(char*)[2] toAutoQualChars(Type t1, Type t2)
     }
     return [s1, s2];
 }
+
+
+/**
+ * For each active modifier (MODFlags.const_, MODFlags.immutable_, etc) call `fp` with a
+ * void* for the work param and a string representation of the attribute.
+ */
+int modifiersApply(TypeFunction tf, void* param, int function(void*, const(char)[]) fp)
+{
+    immutable ubyte[4] modsArr = [MODFlags.const_, MODFlags.immutable_, MODFlags.wild, MODFlags.shared_];
+
+    foreach (modsarr; modsArr)
+    {
+        if (tf.mod & modsarr)
+        {
+            if (int res = fp(param, MODtoString(modsarr)))
```

Comment:
This is a poor solution (even though what you're replacing did the same thing) in that it is pointlessly allocating memory to make a copy of a string literal, and leaking that memory.

Reply:
Call ModToBuffer from here? OK, seems reasonable.